Tests/PlayerTests/Player/PlayerTests.swift

```diff
@@ -50,6 +50,11 @@ final class PlayerTests: XCTestCase {
 		)
 
 		let notificationsHandler = NotificationsHandler.mock()
+		let offlineEngine = OfflineEngine.mock(
+			storage: storage,
```

Comment:
[question] should we have an offline mocked storage? I see we are using `GRDBOfflineStorage`

Reply:
We could mock the storage directly, but we are already mocking the DB that `GRDBOfflineStorage` uses so it is in memory.

So it should not be flaky and slow. 
I would only invest the time to add an `OfflineStorage` mock when we add or modify tests that need it.